copilot/fsd-assessment-store/manifest.yml

```diff
@@ -49,7 +49,7 @@ variables:
   PRIMARY_QUEUE_URL:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-AssessmentImportQueueURL
   DEAD_LETTER_QUEUE_URL:
-    from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-DeadLetterQueueURL
+    from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-AssessmentImportDeadLetterQueueURL
```

Comment:
To match with the change at https://github.com/communitiesuk/funding-service-design-workflows/blob/main/apps/pre-award/copilot/environments/addons/assessment-import-queue.yml#L60